lib/node_modules/@stdlib/math/base/special/cbrt/package.json

```diff
@@ -64,5 +64,81 @@
     "root",
     "power",
     "number"
-  ]
+  ],
+  "__stdlib__": {
+    "scaffold": {
+      "$schema": "math/base@v1.0",
+      "base_alias": "cbrt",
+      "alias": "cbrt",
+      "pkg_desc": "compute the cube root",
+      "desc": "computes the cube root",
```

Comment:
In these descriptions, should we mention whether the package is for single-precision or double-precision numbers?

Reply:
Something like, `compute/computes the cube root of a single/double-precision floating-point number`.

Reply:
No. I think we can rely on scaffolding to insert that language where appropriate. Here, we only want the "essential" bit of the description which distinguishes this package from others.